### ISSUE TITLE
SelectWidget - inconsistent default display

### ISSUE THREAD

**Issue by twMat** [Jermolene/TiddlyWiki5#2329]
The [SelectWidget](http://tiddlywiki.com/#SelectWidget) default display, i.e before any selection is made, should be consistent regardless if the widget is set up for single selection or multiple selections.

Currently, the standard display is a "folded" list but when applying the 
the **multiple** parameter it shows "unfolded". (Note that the *size* parameter does not have to do with this.)

I propose to either always display the list as folded OR, better, to introduce a new parameter to specify if default is folded or unfolded regardless of list type.

**Comment by Jermolene** [comment]
Hi @twMat what you see is the default behaviour of an HTML select element. I think you're asking for a multiple-selection select widget backed by a missing tiddler should display differently than the default; I'm not sure what problem you're trying to solve there? The single vs. multiple variants of the select widget are fundamentally different, and it seems confusing to change their standard behaviour.

**Comment by twMat** [comment]
Thanks for fast reply! It seems I did not explain the problem clearly;

I realized the problem when attempting to use a multiple select in a toolbar; It looks silly because of the open list. Contrast this to a single select where it is "folded" until you actually activate it.

While "single vs multiple" may be fundamentally different in HTML, the SelectWidget UI would allow us to hide such differences and look at it from the user perspective. Here I specifically propose that both variants "look" the same by default - i.e a _folded_ list OR an _unfolded_ list, regardless of variant and before the element is active. Ideally the user could control which folded status to use via a parameter.

**Comment by Jermolene** [comment]
> I realized the problem when attempting to use a multiple select in a toolbar; It looks silly because of the open list

Surely that's just because a multiple selection select widget is taller than the toolbar? The usual way around that would be to put it in a dropdown.

Otherwise, can't you just use CSS to constrain the height of your multiple selection select widget to a single line? But I'd have thought the user experience would be terrible: for a multiple selection box to be usable one needs to see several rows at once.

In other words, I think you're asking for a multiple selection select widget that allows for multiple selections but only displays as a single line, like the single selection select widget. That's impossible; like asking for a skinless apple to have the same peel as an orange.

**Comment by twMat** [comment]
Yes, it is possible to circumvent with dropdowns or CSS. 

>But I'd have thought the user experience would be terrible: for a multiple selection box to be usable one needs to see several rows at once.

It makes little sense to me that - before any selection is made - one variant should display its list but the other shouldn't.

That said, I'm beginning to think my use case is not a typical one; I think of the select widget primarily as a tool to _set values_ rather than to _display_ the widget label(s). This is why it makes more sense to me to have the widget fold up.

Because my use case and perception of the tool is probably not the "standard", I'm closing the issue.
